dali/pipeline/operators/optical_flow/optical_flow.h

```diff
@@ -118,13 +118,15 @@ class OpticalFlow : public Operator<Backend> {
       sequence_sizes_[i] = shape[i][0];
     }
 
-    DALI_ENFORCE([&]() -> bool {
-        for (const auto &seq : shape) {
-          if (seq[1] != frames_height_ || seq[2] != frames_width_ || seq[3] != depth_)
-            return false;
-        }
-        return true;
-    }(), "Width, height and depth must be equal for all sequences");
+    for (const auto &seq : shape) {
+      DALI_ENFORCE(seq[1] != frames_height_ || seq[2] != frames_width_ || seq[3] != depth_,
+              "Width, height and depth for Optical Flow calculation must be equal for all sequences.");
+    }
+
+    for (auto sz : sequence_sizes_) {
+      DALI_ENFORCE(sz < 2, 
+        "Optical Flow operator error: Number of frames in every sequence must be greater or equal to 2.");
```

Comment:
```suggestion
        ""At least two frames are necessary to calculate optical flow!");
```
It's shorter and explains the problem better.